src/OmniSharp.Roslyn.CSharp/Services/Refactoring/V2/CodeFixesForProjects.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Composition;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace OmniSharp.Roslyn.CSharp.Services.Refactoring.V2
+{
+    [Shared]
+    [Export(typeof(CodeFixesForProjects))]
+    public class CodeFixesForProjects
+    {
+        private readonly ConcurrentDictionary<string, IEnumerable<CodeFixProvider>> codeFixCache = new ConcurrentDictionary<string, IEnumerable<CodeFixProvider>>();
```

Comment:
nit: you left out the `_` prefix for field names that you used elsewhere.

Reply:
We are all waiting https://github.com/OmniSharp/omnisharp-vscode/issues/1726 😄